internal/pkg/mapper/relations/load.go

```diff
@@ -61,13 +61,7 @@ func (m *relationsMapper) linkRelations(object model.ObjectWithRelations, event
 				thisSideKey.Desc(),
 				relation.Type(),
 			)
-
-			if relation.IgnoreMissingOtherSide() {
-				m.Logger.Warn(`Warning: `, err)
-			} else {
-				errors.Append(err)
-			}
-
+			m.Logger.Warn(`Warning: `, err)
```

Comment:
Ak neexistuje objekt, na ktory odkazuje `Relation`, takto bude vzdy `Warning`, bude to takto jednoduchsie, ... a ako rozmyslam nad vsetkymi pripadmi, tak nikdy nebudeme chciet uplne zablokovat dalsiu pracu uzivatela.